app/views/volunteers/_profile.html.erb

```diff
@@ -0,0 +1,51 @@
+<div>
+    <div>
+        <h4>My Profile</h4>
+    </div>
+    <div>
+        <img src="<%= @volunteer.picture %>" height=50px>
+        <h5>Name:</h5> <p><%= @volunteer.first_name %> <%= @volunteer.last_name %></p>
+        <h5>Email:</h5> <p><%= @volunteer.email %></p>
+        <h5>Phone:</h5> <p><%= @volunteer.mobile_phone %></p>
+        <h5>Location:</h5> <p><%= @volunteer.city %>, <%= @volunteer.state %> </h5>
```

Comment:
As mentioned above, you can consider replacing the `<h5>` headings with `f.label`.